.github/workflows/npc_db_validation.yml

```diff
@@ -6,6 +6,9 @@ name: Validate NPC Scripts and DB Changes
 
 on:
   push:
+    branches:
+      - master
+  pull:
     paths:
       # Always trigger all Github Actions if an action or something CI related was changed
       - '.github/workflows/**'
```

Comment:
```suggestion
on: [push, pull_request]
  branches:
    - master
  paths:
      # Always trigger all Github Actions if an action or something CI related was changed
    - '.github/workflows/**'
```

Reply:
`pull:` isn't a valid trigger - silly inline suggestions wouldn't let me change the indent of the other path lines o.O